uint64 timestamp              # time since system start (microseconds)
uint64 timestamp_requested

Parameter parameter

uint8 RESULT_SET_SUCCESS             = 0
uint8 RESULT_SET_FAILED              = 1
uint8 RESULT_ERROR_INVALID_PARAMETER = 2
uint8 result

uint8 ORB_QUEUE_LENGTH = 8
